docs/host/host.md

```diff
@@ -23,6 +23,16 @@ Because Harvester is built on top of Kubernetes and uses etcd as its database, t
 
 For admin users, you can click **Enable Maintenance Mode** to evict all VMs from a node automatically. It will leverage the `VM live migration` feature to migrate all VMs to other nodes automatically. Note that at least two active nodes are required to use this feature.
 
+If you want to individually force a shutdown of a VM instead of migrating it to another node, add the label `harvesterhci.io/maintain-force-shutdown-strategy` and one of these values to those VMs:
```

Comment:
This label `harvesterhci.io/maintain-force-shutdown-strategy`, is binding `forces shutdown` and `later restart option` into one label,

Could it be renamed as `harvesterhci.io/node-maintenance-mode-strategy`, and the following options:

`RestartOnEnable` -> `ShutdownAndRestartAfterEnable`

`RestartOnDisable` -> `ShutdownAndRestartOnDisable`

`StayOff` -> `Shutdown`

New option: `LiveMigrate`


And, the `LiveMigrate` is the default value, as long as user set/not set this label on the VM, unless one of the other three options is explicitly hit.

Reply:
I've added the suggestions to a HEP for this feature. See https://github.com/harvester/harvester/pull/5984